using-mobile-extensions/adobe-target-vec.md

```diff
@@ -0,0 +1,144 @@
+# Adobe Target - VEC
+This extension enables Target VEC (Visual Experience Composer) functionality. The Target VEC for Native Mobile Apps lets you create activities and personalize content on native mobile apps in a do-it-yourself fashion without continuous development dependencies and app-release cycles.
+
+{% hint style="info" %}
+Review the following Target VEC functionality documentation for context before implementation at these links:
```

Comment:
Please change to:

"For more information about Target VEC, see the following links in the Target VEC documentation:"